yoga/YGEnumsPrint.h

```diff
@@ -0,0 +1,359 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#pragma once
+
+#include "YGMacros.h"
+#include "Yoga.h"
+
+YG_EXTERN_C_BEGIN
+
+void YGLogAlign(YGLogLevel logLevel, const char * param, YGAlign value){
+  char * text = "undefined";
+  switch(value){
+    case YGAlignAuto:
+      text = "auto";
+      break;
+    case YGAlignFlexStart:
+      text = "flex-start";
+      break;
+    case YGAlignCenter:
+      text = "center";
+      break;
+    case YGAlignFlexEnd:
+      text = "flex-end";
+      break;
+    case YGAlignStretch:
+      text = "stretch";
+      break;
+    case YGAlignBaseline:
+      text = "baseline";
+      break;
+    case YGAlignSpaceBetween:
+      text = "space-between";
+      break;
+    case YGAlignSpaceAround:
+      text = "space-around";
+      break;
+  }
+  if(param == NULL) {
+    YGLog(logLevel, text);
```

Comment:
I also thought of this, but this makes the callside a bit more boilerplate, not sure if we want this.